src/SonarScanner.MSBuild.PreProcessor/AnalysisConfigProcessing/AnalysisConfigGenerator.cs

```diff
@@ -23,12 +23,10 @@
 using System.Linq;
 using SonarScanner.MSBuild.Common;
 
-namespace SonarScanner.MSBuild.PreProcessor;
+namespace SonarScanner.MSBuild.PreProcessor.AnalysisConfigProcessing;
```

Comment:
I made a directory and put all of the classes that have to do with the AnalysisConfig in there.